tests/values.go

```diff
@@ -94,6 +94,12 @@ type CoderdValues struct {
 	Proxy                         *CoderdProxyValues                         `json:"proxy" yaml:"proxy"`
 	ReverseProxy                  *CoderdReverseProxyValues                  `json:"reverseProxy" yaml:"reverseProxy"`
 	NetworkPolicy                 *CoderdNetworkPolicyValues                 `json:"networkPolicy" yaml:"networkPolicy"`
+	ClientTls                     *CoderdClientTlsValues                     `json:"clientTls" yaml:"clientTls"`
+}
+
```

Comment:
needs corresponding tests, @ericpaulsen is working on these for something else and might be able to help?

Reply:
done!